mlrun/utils/http.py

```diff
@@ -0,0 +1,136 @@
+import time
+
+import requests
+import requests.adapters
+import urllib3.util.retry
+
+from ..config import config
+from . import logger
+
+DEFAULT_RETRY_COUNT = 3
+DEFAULT_RETRY_BACKOFF = 1
```

Comment:
Now that it's a centralized config and used in several places, I think it makes sense to put these parameters in the general config. This will enable fine-tuning in the field if needed.

Also - why did you rename `HTTP_RETRY_BACKOFF_FACTOR` back to `HTTP_RETRY_BACKOFF`?